docker-compose.yml

```diff
@@ -23,15 +23,15 @@ version: "3.9"
 
 services:
   db:
-    image: couchbase/server-sandbox:7.0.0
+    image: couchbase/server-sandbox:7.1.0
     ports:
-      - "8091-8095:8091-8095"
+      - "8000-8005:8000-8005"
       - "9102:9102"
-      - "11210:11210"
+      - "11200:11200"
```

Comment:
again, I thought I left a comment about this... what is this change for, and will it break non-capella usage?